vignettes/submodel_documentation_overview.Rmd

```diff
@@ -0,0 +1,394 @@
+- `fecundity_lookup`: [Roni and Quinn 1995](https://reorienting-to-recovery.s3.us-west-1.amazonaws.com/RoniandQuinn1995.pdf) logic was used to map age to size. Then we scaled fecundity by size using logic from [Malick et all 2023.](https://www.x-mol.net/paper/article/1664889742509719552)
+- `..surv_egg_to_fry_int`: Intercept, source: Calibration
+- `mean_egg_temp_effect`: Fall and Spring Run values are the same and were estimated by C. Hammersmark (CBEC ECOengineering Inc.). These Fall and Spring Run values were calculated by taking the mean of dry and wet egg temp effects from previous model. Winter run values were calculated through model calibration.
+- `.surv_egg_to_fry_proportion_natural`: Coefficient for proportion_natural variable, source: Chilcote et al. 2011
+- `.surv_egg_to_fry_scour`: Coefficient for scour variable, source: Montgomery et al. 1996
+
+## Juvenile 
+
+### Growth 
+
+Juvenile growth is based on bioenergetics transition matrices that show Chinook growth rates in floodplain and inchannel habitat. Growth varies by habitat with higher growth rates in floodplain habitat and lower in inchannel habitat. 
+
+- data: `prey_density` (max, high, medium, or low)
+- data: `prey_density_delta` (max, high, medium, or low)
+
+*Source* 
+Empirical prey density data were aggregated from multiple authors and across many watersheds. For each dataset, the habitat type was assigned. Specific methods and supporting documents for data aggregation and decisions in every watershed are provided by author. See [preyDataProcessing](https://github.com/CVPIA-OSC/preyDataProcessing) for more details. Data processed in bioenergetics model which provides distinct growth rates for max, high, medium, and low prey. Model inputs are prey data quantities for each tributary and the delta. model is calibrated to medium prey density on all watersheds. 
+
+### Movement 
+
+#### Route 
+The route function determines if juveniles stay in their natal tributary, are detoured to a bypass, or out migrate during a simulated month. Routing is based on amount of habitat, and flows. 
+
+The route function takes in the following data objects: 
+
+- data: `inchannel_habitat`
+- data: `floodplain_habitat` 
+- data: `prop_pulse_flows`
+- data: `proportion_flow_bypass`
+- data: `territory_size`
+- data: `freeport_flows`
+- data: `vernalis_flows`
+- coefficient: `.pulse_movement_intercept`
+- coefficient: `.pulse_movement_proportion_pulse`
+- coefficient: `.pulse_movement_medium`
+- coefficient: `.pulse_movement_large`
+- coefficient: `.pulse_movement_vlarge`
+- coefficient: `.pulse_movement_medium_pulse`
+- coefficient: `.pulse_movement_large_pulse`
+- coefficient: `.pulse_movement_very_large_pulse`
+- filling ruleset, defaults to regional: `fill_regional`
+
+and it is input to the `fall_run_model` as part of the params list. 
+
+
+*Sources*
+- `prop_pulse_flows`: Flow data from DSMflow package. 
```

Comment:
for some reason this isn't showing as a list, so will need to add a new line above I think to fix that